contrib/ACME-CA.cfg

```diff
@@ -0,0 +1,14 @@
+organization = "ACME Corp."
+cn = "ACME CA"
+uri = "http://www.example.com/"
+email = "admin@example.com"
+crl_dist_points = "http://www.example.com/pki/"
+serial = 1
+crl_number = 1
+path_len = 1
+activation_date = "2017-08-01 00:00:00Z"
+expiration_date = "2047-08-01 00:00:00Z"
```

Comment:
To discourage use in production what do you think about setting the default value for activation as the date you run the script and expiration as 30 days later?

Reply:
Have something like ACME-CA.cfg.in and then process it with meson? My worry with that would be that when you're testing libjcat might not be the same time that you're building jcat. Or you mean something like injecting the dates into the cfg using the shell script when it builds the cert? Maybe the shell script actually wants to be python if we're doing anything remotely clever...

Reply:
I was thinking injecting dates when building certs.

Reply:
Something like this?